librz/analysis/flirt.c

```diff
@@ -1389,12 +1390,15 @@ static RzFlirtNode *flirt_parse(const RzAnalysis *analysis, RzBuffer *flirt_buf)
 
 	if (header->features & IDASIG__FEATURE__COMPRESSED) {
 		if (version == 5) {
```

Comment:
I guess `>= 5`

Reply:
No, just version 5 is compressed without header.

Reply:
if there is a version 6 or higher, then is better to have a check on this